app-front/src/components/common/SelectOperator.tsx

```
@@ -3,31 +3,29 @@ import { Config } from "@/config";
-      }}
-    >
-      {data?.map((d, i) => (
-        <option key={i} value={d.id}>
-          {d.name}
-        </option>
-      ))}
+      }}      
+    > 
+      { data &&
+        data.data.map((d, i) => <option key={i} value={d.id}>{d.name}</option>)
+      }
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)